recipes/aptools/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.2.4" %}
+
+package:
+  name: aptools
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/AngelFP/APtools/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d382b6700668194d099c014b63c8e6d34cf0cb26cccc0a85daeef8a5883de8d2
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Reply:
Hi @ocefpaf :wave:
I'm curious about this one as I've previously recommended to drop the `--no-deps --no-build-isolation` flags with the understanding that they were already configured on `conda-forge`.

My go-to reference which I usually recommend is to follow the example recipe:
https://github.com/conda-forge/staged-recipes/blob/b9fe52599c957d19c9cdbf926d19a66be7460cdb/recipes/example/meta.yaml#L38


Is that still good advice or should we be recommending to add those flags?

Reply:
The KB seems to also follow the simple version implicitly in some examples:
https://conda-forge.org/docs/maintainer/knowledge_base.html

Will stand by for advise :)

Reply:
From gitter (thanks to `jakirkham`) it seems you only need to use `-vv` here as the rest are set by `conda-forge` and this recipe doesn't use `outputs`.

The underlying issue with `outputs` - https://github.com/conda/conda-build/issues/3993

Reply:
Confirmed via chat:
https://matrix.to/#/!SOyumkgPRWoXfQYIFH:matrix.org/$17024301973176NcLkK:matrix.org?via=matrix.org&via=gitter.im&via=cadair.com

Conda adds automatically
```
... -vv --no-deps --no-index --no-build-isolation --ignore-installed --cache-dir pip_cache
```
unless recipe `outputs` are used.
https://github.com/conda-forge/implicit-feedstock/blob/3832f659196b1aae468470f1bda12b59e268ced4/recipe/meta.yaml#L42

Reply:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

Reply:
In theory conda-build has those flags but it doesn't hurt to add them in fact latest version of grayskull is adding them by default. The reason is to avoid bugs like in the multiple output recipe where the flags were not initialized.